svthalia/concrexit
# Make Slide.url fall back to the linked event

Closes #968 



### Summary
Slide.url is now a pseudo-property that checks either `custom_url` or `event`, and links to the event if `custom_url` is not set.

### How to test
Steps to test the changes you made:
1. Create a slide, give it an event but no link
2. View the homepage with the slide enabled
3. Observe that the slide now links to the relevant event
